components/custom_status/emoji/index.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/common';
+
+import {GlobalState} from 'types/store';
+
+import CustomStatusEmoji from './custom_status_emoji';
+
+function mapStateToProps(state: GlobalState) {
```

Comment:
Instead of passing user in props we can use do this
`const currentUser = useSelector((state: GlobalState) => getCurrentUser(state));`  
directly in functional component